nixos/modules/services/misc/ombi.nix

```diff
@@ -0,0 +1,76 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let cfg = config.services.ombi;
+
+in {
+  options = {
+    services.ombi = {
+      enable = mkEnableOption "Ombi";
+
+      dataDir = mkOption {
+        type = types.str;
+        default = "/var/lib/ombi/.config/Ombi";
+        description = "The directory where Ombi stores its data files.";
+      };
+
+      port = mkOption {
+        type = types.int;
+        default = 5000;
+        description = "The port for the Ombi web interface.";
+      };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Open ports in the firewall for the Ombi web interface.";
+      };
+
+      user = mkOption {
+        type = types.str;
+        default = "ombi";
+        description = "User account under which Ombi runs.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "ombi";
+        description = "Group under which Ombi runs.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.tmpfiles.rules = [
+      "d '${cfg.dataDir}' 0700 ${cfg.user} ${cfg.group} - -"
+    ];
+
+    systemd.services.ombi = {
```

Comment:
What about users interested in [alternate databases](https://docs.ombi.app/info/alternate-databases/)?

Reply:
Well there is nothing stopping them from creating the file manually. I guess we could also create an option for that, but it would be a little more complicated. However you think it's best.

Reply:
If the user can place the file in the state directory that sounds good enough. Someone could always add later if needed.